src/antsibull/cli/antsibull_build.py

```diff
@@ -207,6 +217,9 @@ def parse_args(program_name: str, args: list[str]) -> argparse.Namespace:
                                          help='Directory to write .build and .deps files to,'
                                          ' as well as changelog and porting guide if applicable.'
                                          '  Defaults to --data-dir')
+    build_write_data_parser.add_argument('--tags-file', action='store_true',
```

Comment:
I think this is the wrong place. For example the `new-ansible` command doesn't need this. Also the explanation should be different for `prepare`/`single`/`multiple` and `rebuild-single`.

Reply:
https://github.com/ansible-community/antsibull/pull/473/commits/53559b99525d19ae85b56ae9ec51e42f3450c057